modules/artifact/src/test/java/org/mule/module/artifact/classloader/ClassLoaderLookupPolicyTestCase.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright (c) MuleSoft, Inc.  All rights reserved.  http://www.mulesoft.com
+ * The software in this package is published under the terms of the CPAL v1.0
+ * license, a copy of which has been included with this distribution in the
+ * LICENSE.txt file.
+ */
+
+package org.mule.module.artifact.classloader;
+
+import org.mule.tck.junit4.AbstractMuleTestCase;
+import org.mule.tck.size.SmallTest;
+
+@SmallTest
+public class ClassLoaderLookupPolicyTestCase extends AbstractMuleTestCase
+{
+
+    /*
```

Comment:
I don't know what happened with this code... maybe I didn't pushed my latest changes before I created the PR